syntax = "proto3";

import "google/protobuf/empty.proto";

service PGService {
  rpc TransactionPerform(TransactionPerformRequest)
      returns (google.protobuf.Empty) {};
}

message TransactionPerformRequest {
  string userID = 1;
  float amount = 2;
  int32 type = 3;
}
